Makefile.toml

```diff
@@ -43,3 +42,14 @@ dependencies = [
     "clippy",
     "docs"
 ]
+
+# testing
+[tasks.test]
+command = "cargo"
+args = ["test", "--all-features", "--workspace", "--", "--nocapture"]
+
+# docker
+[tasks.docker-build-node]
+command = "docker"
+args = ["build", "-f", "node/Dockerfile", "-t", "miden-node-image", "."]
```

Comment:
This is not setting the `ENV` vars:

```
time="2024-03-14T13:29:04+01:00" level=warning msg="missing \"CREATED\" build argument. Try adding \"--build-arg CREATED=<VALUE>\" to the command line"
time="2024-03-14T13:29:04+01:00" level=warning msg="missing \"VERSION\" build argument. Try adding \"--build-arg VERSION=<VALUE>\" to the command line"
time="2024-03-14T13:29:04+01:00" level=warning msg="missing \"COMMIT\" build argument. Try adding \"--build-arg COMMIT=<VALUE>\" to the command line"
```

Reply:
Indeed, I am curious from what data you would want to populate these fields ?

Reply:
I added the version and created from the crate version and `date` shell command. 

I am not sure how you would like to get the commit info.

Reply:
The `COMMIT` should have the git commit.
The `VERSION` should have the node's software version.
The `CREATED` should have the created timestamp.